docs/WORDLIST.md

```diff
@@ -276,10 +276,10 @@ RTS instruction at the end of each word.
 | WORDSIZE | `wordsize` | Tali Forth | 32 | **auto** |
 | XOR | `xor` | ANS core | 17 | **auto** |
 | ZERO | `0` | Tali Forth | 6 | **auto** |
-| ZERO_EQUAL | `0=` | ANS core | 19 | **auto** |
+| ZERO_EQUAL | `0=` | ANS core | 17 | **auto** |
```

Comment:
saves a couple bytes and cycles for each (one path of equal stays the same iirc)